deps/rabbit/src/rabbit_db_cluster.erl

```diff
@@ -295,10 +295,7 @@ members_using_mnesia() ->
     rabbit_mnesia:members().
 
 members_using_khepri() ->
-    case rabbit_khepri:locally_known_nodes() of
-        []      -> [node()];
-        Members -> Members
-    end.
+    rabbit_khepri:locally_known_nodes().
```

Comment:
Could you please add a comment about the difference of behavior/return value compared to Mnesia?